src/axom/inlet/DocWriter.hpp

```diff
@@ -0,0 +1,35 @@
+#include <string>
+#include <vector>
+#include <fstream>
+
+#include "axom/sidre.hpp"
+
+#ifndef INLET_DOCWRITER_HPP
+#define INLET_DOCWRITER_HPP
+
+namespace axom
+{
+namespace inlet
+{ 
+
+class DocWriter {

```

Comment:
Please rename this class to SphinxDocWriter and inherit from a new pure virtual class called DocWriter that only defines the constructor and writeDocuments.  This way users can create their own class if they wish that acts differently.

Reply:
Also add a setter for the sidre root group that inlet can set in the Inlet::registerDocWriter function